build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
@amila-ku, as suggested by @objectiser in #767, could you please use `registry.redhat.io` instead of `registry.access.redhat.com`?

Reply:
ok, but when i tested i could not use without authenticating. I will change and check it out

Reply:
e2e failures due to  unauthorized.

```
Step 1/7 : FROM registry.redhat.io/ubi8/ubi
Get https://registry.redhat.io/v2/ubi8/ubi/manifests/latest: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication
##[error]Makefile:65: recipe for target 'docker' failed
make: *** [docker] Error 1
##[error]Process completed with exit code 2.
```

Reply:
Thanks for testing, @amila-ku! I think we are safe to go with `registry.access.redhat.com` then.